workers/unity/Assets/Playground/Config/PlayerTemplate.cs

```diff
@@ -22,29 +22,26 @@ public static class PlayerTemplate
                     $"Expected an attribute that is not \"{WorkerUtils.UnityClient}\" but none was found.");
             }
 
-            var transform =
-                TransformInternal.Component.CreateSchemaComponentData(new Location(),
-                    new Quaternion { W = 1, X = 0, Y = 0, Z = 0 }, new Velocity(0.0f, 0.0f, 0.0f), 0, 0.0f);
+            const string CharacterType = "Character";
```

Comment:
I don't see a point in having a string constant if it's just used once...

Reply:
thanks for the spot - seems to have carried forward from a merge, will remove